pulsar/internal/lookup_service.go

```diff
@@ -325,9 +325,11 @@ func (h *httpLookupService) GetPartitionedTopicMetadata(topic string) (*Partitio
 
 	path := fmt.Sprintf(format, GetTopicRestPath(topicName))
 
+	h.log.Infof("Fetch topic{%s} partitioned metadata %s", topic, path)
```

Comment:
This we should maybe leave as debug level

Reply:
I have removed this line since I was trying to debug the reason why the tests are failed.